test/test_components.py

```diff
@@ -1140,7 +1143,10 @@ def check_odd(array):
             return array % 2 == 0
 
         iface = gr.Interface(check_odd, "numpy", "numpy")
-        self.assertEqual(iface.process([[2, 3, 4]])[0], {"data": [[True, False, True]]})
+        print(iface.process([{"data": [[2, 3, 4]]}]))
```

Comment:
nit: Think we should remove this print statement prior to merge

Reply:
ops